src/libraries/Microsoft.Extensions.Primitives/src/StringTokenizer.cs

```diff
@@ -8,7 +8,7 @@
 namespace Microsoft.Extensions.Primitives
 {
     /// <summary>
-    /// Tokenizes a <see cref="string"/> into <see cref="StringSegment"/>s.
+    /// Tokenizes a <see cref="string"/> into <see cref="StringSegment"/> segments.
```

Comment:
Would you accept this?

```suggestion
    /// Tokenizes a <see cref="string"/> into <see cref="StringSegment"/>StringSegments</see>.
```